pkgs/top-level/all-packages.nix

```diff
@@ -29014,7 +29014,11 @@ with pkgs;
     withNetworkd = true;
     withLibidn2 = true;
   };
-
+  # remove when util-linux 2.39.3 comes out
+  systemdBcachefs = systemd.override {
```

Comment:
nitpick: I think even for overrides we can already use pkgs/by-name

Reply:
@Mic92 Yeah, I just did a quick and dirty change until the main util-linux gets a patch. I was considering moving tools in to by-name.